shenyu-admin/src/main/java/org/apache/shenyu/admin/disruptor/executor/RegisterServerConsumerExecutor.java

```diff
@@ -64,8 +58,7 @@ public void run() {
     private boolean isValidData(final Object data) {
         if (data instanceof URIRegisterDTO) {
             URIRegisterDTO uriRegisterDTO = (URIRegisterDTO) data;
-            return Objects.nonNull(uriRegisterDTO.getPort())
-                    && StringUtils.isNoneBlank(uriRegisterDTO.getAppName(), uriRegisterDTO.getHost());
+            return StringUtils.isNotBlank(uriRegisterDTO.getContextPath());
```

Comment:
why this Judgment？

Reply:
When there is no upstream service, there are only the contextPath and rpcType attributes in the URIRegisterDTO. rpcType is added this time, URIRegisterExecutorSubscriber.findService needs to use.

Reply:
The status of the upstream service is set in CommonUpstreamUtils.